cognite/client/_api_client.py

```diff
@@ -109,11 +114,8 @@ def __init__(
             session=session,
         )
 
-        self._CREATE_LIMIT = 1000
-        self._LIST_LIMIT = 1000
-        self._RETRIEVE_LIMIT = 1000
-        self._DELETE_LIMIT = 1000
-        self._UPDATE_LIMIT = 1000
+    def _override_request_limits(self) -> None:
+        """Allow sub-APIs to more easily override specific limits without the full super().__init__ dance"""
```

Comment:
This would make more sense if you made the limit attributes private in APIClient, and instead had `_override_request_limits` return an object containing the override key-value pairs. But that sounds like a lot of hassle for doing something which already has a standard approach.